samples/servicecontrol/events-subscription/sample.md

```diff
@@ -72,7 +72,7 @@ Retries are disabled in the sample for simplicity; messages are immediately move
 
 snippet: DisableRetries
 
-The `MessageFailed` event is published for any standard NServiceBus endpoint that is monitored by ServiceControl.
+The `MessageFailed` event is published for any NServiceBus endpoint that is monitored by ServiceControl.
```

Comment:
What does it mean to publish events for the endpoints monitored? Isn't they key think that we publish the event when we detect a failed message?